src/ralph/data_center/tests/test_publishers.py

```diff
@@ -0,0 +1,38 @@
+from ralph.data_center.publishers import _get_host_data
+from ralph.security.tests.factories import SecurityScanFactory
+from ralph.tests import RalphTestCase
+from ralph.virtual.tests.factories import VirtualServerFactory
+
+
+class PublisherTests(RalphTestCase):
+
+    def test_get_host_data(self):
```

Comment:
please rename it to `test_get_host_data_after_deleting_securityscan`

Reply:
Sure.